frontend/app/src/components/history/events/HistoryEventsView.vue

```diff
@@ -332,6 +339,13 @@ async function redecodePageTransactions(): Promise<void> {
   await fetchUndecodedTransactionsStatus();
   await fetchData();
 }
+
+function removeIdentifierParam() {
+  const query = { ...route.query };
+  delete query.identifiers;
+  router.replace({ query });
```

Comment:
Like I mentioned on discord it would make sense to use `push` here just to keep it in history in case you want to navigate back.